### ISSUE TITLE
There is no way to enable recycle for user homes in OMV 5

### ISSUE THREAD

**Issue by alanalvarado** [openmediavault/openmediavault#1165]
**Is your feature request related to a problem? Please describe.**
There is no way to enable recycle for user homes in OMV 5

**Describe the solution you'd like**
a button to enable it inside the samba configuration module

**Describe alternatives you've considered**
I added vfs object in extra options,
vfs objects = recycle
recycle:repository = .recycle/%U
recycle:keeptree = yes
recycle:versions = yes
recycle:exclude_dir = .recycle
but [homes] share overrides this option by adding an empty line with vfs object =

I also added in extra options an include 
include = /etc/samba/homes.conf
to external homes.conf file with the next content
[homes]
recycle:repository = .recycle/%U 

but its also loaded before [homes] in main smb.conf and gets overridden too.

**Comment by alanalvarado** [comment]
BTW I'm seeing features that require WEB GUI modifications won't be fixed. But I think at least the empty line vfs object = could be removed from the script that creates the home share. and then use the global vfs object, that could probably work. Thanks in advance.

**Comment by alanalvarado** [comment]
Ok I just tried this 
`omv-env set OMV_SAMBA_HOMES_VFSOBJECTS "recycle"
`

and then
`omv-salt deploy run samba
`

then enabled again the home directories share but no luck the vfs line is still empty

the file [homes.j2](https://github.com/openmediavault/openmediavault/blob/master/deb/openmediavault/srv/salt/omv/deploy/samba/files/homes.j2)

contains this line 
`{%- set vfs_objects = salt['pillar.get']('default:OMV_SAMBA_HOMES_VFSOBJECTS', '') -%}
`
which apparently sets the variable again to empty because of the last '' after the ,

I modified this file inside /srv/salt/omv 

and added recycle inside the '' and it works.

Anyway this was very hard to achieve and took me around 10 hours to look through forums and finding through files and documentation.

isn't there a simpler solution? also editing homes.j2 locally won't be resilient on subsequent updates.
